ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Generators.hs

```diff
@@ -175,8 +176,14 @@ instance {-# OVERLAPPABLE #-} Arbitrary a
 -- make sure to not generate those queries in combination with
 -- 'ShelleyNodeToClientVersion1'.
 instance CanMock era
-      => Arbitrary (WithVersion ShelleyNodeToClientVersion (SomeSecond Query (ShelleyBlock era))) where
+      => Arbitrary (WithVersion ShelleyNodeToClientVersion (SomeSecond BlockQuery (ShelleyBlock era))) where
   arbitrary = do
       query@(SomeSecond q) <- arbitrary
       version <- arbitrary `suchThat` querySupportedVersion q
       return $ WithVersion version query
+
+instance Arbitrary (WithVersion ShelleyNodeToClientVersion (SomeSecond BlockQuery (ShelleyBlock era)))
```

Comment:
Same question as in my comment on the corresponding instance in `ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/Generators.hs`